bigframes/core/compile/compiled.py

```diff
@@ -1034,6 +1057,9 @@ def _to_ibis_expr(
         return table
 
     def filter(self, predicate: ex.Expression) -> OrderedIR:
+        if any(map(is_window, map(self._get_ibis_column, predicate.unbound_variables))):
+            # ibis doesn't support qualify syntax
```

Comment:
Might be worth an issue, too. I think there is some support in ibis for `having`, for example (though it's not really possible with the pandas API) if I remember correctly from my work on F1. https://ibis-project.org/tutorials/ibis-for-sql-users#having-clause

Reply:
created bug and referenced in comment